src/diracx/cli/internal.py

```diff
@@ -104,7 +104,8 @@ def add_user(
     typer.echo(f"Writing back configuration to {yaml_path}", err=True)
     config_data = json.loads(config.json(exclude_unset=True))
     yaml_path.write_text(yaml.safe_dump(config_data))
-    repo.index.add([yaml_path])
+    # MacOS adds a "/private" in front, so we need to resolve the path when adding
+    repo.index.add([yaml_path.resolve()])
```

Comment:
Maybe changing line 103 to `yaml_path = "default.yml"` would be preferable? (And the same in `generate_cs`)